wgsl/index.bs

```diff
@@ -10541,196 +10541,452 @@ See [[#function-calls]].
 ## Logical Built-in Functions ## {#logical-builtin-functions}
 
 <table class='data'>
-  <thead>
-    <tr><th>Parameterization<td style="width:40%;">Overload<td>Description
-  </thead>
-  <tr algorithm="vector all">
+  <tr>
+    <th scope="row" style="width:10%">Parameterization
     <td>
-    <td>`@const fn`<br>`all`(|e|: vec|N|&lt;bool&gt;) -> bool
+  <tr algorithm="vector all">
+    <th scope="row">Overload
+    <td><xmp highlight=rust>@const fn all(|e|: vec|N|<bool>) -> bool</xmp>
+  <tr>
+    <th scope="row">Description
     <td>Returns true if each component of |e| is true.
+</table>
 
+<table class='data'>
+  <tr>
+    <th scope="row" style="width:10%">Parameterization
+    <td>
   <tr algorithm="scalar all">
-    <td>|e|: bool
-    <td>`@const fn`<br>`all(`|e|`)` -> bool
+    <th scope="row">Overload
+    <td class="nowrap"><xmp highlight=rust>@const fn all(|e|: bool) -> bool</xmp>
```

Comment:
I actually find it harder to figure which builtin is being described now. Previously it wasn't amazing but, the names generally lined up with little intervening text. Now it feel harder to spot the actual function. Maybe have an extra row with merged columns that is the name of the builtin.

E.g.
```
<table class='data'>
  <tr><th>**all**
  <tr><td>**Description**
           <td>Does all
   <tr><td>**Overload**
           <td>`@const fn all(e : bool) -> bool`
   <tr><td>**Parameterization**
           <td>
</table>
```

Reply:
Agreed.  This is my main issue with this way of presenting.

In this rendering, there's a lot of space and visual attention given to Parameterizaion, Overload, Description.
Meanwhile it's very hard to pick out which builtin is being described.

I see the benefit of allowing more breathing space for function descriptions.   It makes room for more examples, especially over time.

I think a good model are the Unix manual pages for library functions. 
E..g. try `man 3 frexp` and see its structure:  NAME, SYNOPSIS, DESCRIPTION, and various patterns run from there,  including SPECIAL VALUES, SEE ALSO, STANDARDS


And then that reminds me that those literally used to be paper _pages_ in a manual that you held in your hands (mano).  

If we made each function their 3rd level section, then that solves two problems:  
- being able to quickly scan the list of functions to see what is available, and
- visually highlighting and clearly separating each function from the next.


Can I suggest doing one or two functions at a time on the next round?  :-)

Reply:
PTAL.